srcpkgs/NetworkManager-openconnect/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'NetworkManager-openconnect'
 pkgname=NetworkManager-openconnect
-version=1.2.6
-revision=4
+version=1.2.10
+revision=1
+archs="i686 x86_64 x86_64-musl"
```

Comment:
this isn't how to disable something temporarily, use `broken`. but it would be better to figure out the error

Reply:
Broken blocks all installs independent of architecture, right?

Reply:
you put it in an if/case block to make it conditional